### DISCUSSION TITLE
run bash command after git clone

### DISCUSSION THREAD

**Discussion by benjaminpreiss** [coollabsio/coolify#4239] [Feature Requests]
I want to deploy openreplay on coolify using secrets managed by infisical.
openreplay requires a setup command right after git clone and before docke-compose build, that loads a .env file.
The secrets lie within infisical.

So, what I want to do is:

- git clone
- run infisical to generate .env file: `infisical export --token=$(infisical login --method=universal-auth --client-id=***** --client-secret=***** --silent --plain) --projectId="*****" --format=dotenv > .env`
- run [load.sh](https://github.com/frontline-hq/openreplay/blob/main/scripts/docker-compose/load.sh) command that loads .env file and generates docker-compose file: `./load.sh`
- docker-compose build etc...

Currently, I have a fork of openreplay that I use: [https://github.com/frontline-hq/openreplay](https://github.com/frontline-hq/openreplay). I need to use this fork, because the original openreplay doesnt allow loading from a .env currently.

It would be great if coolify would allow running a custom command (with access to env variables) after git clone.

**Comment by viisrael** [comment]
Hi, @benjaminpreiss . Did you install OpenReplay on coolify?
